test/functional/fixtures/es_archiver/discover/data.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
I think this might have been added by mistake. The data is usually gzipped

Reply:
No, it's possible, zipped and unzipped, and I personally prefer a unzipped version for little data, because then I've got instant access to the contents of this file, and it's sometimes useful to have a quick look. But would be no big deal for me to zip it, if you prefer that :)

Reply:
Whichever you prefer. Mostly making sure its not my own mistake.